### PR TITLE
Fix xcm precompiles minor bugs

### PR DESCRIPTION
**Pull request by akru**
**Pull Request Summary**

**Check list**
- [ ] added unit tests
- [ ] updated documentation

### CODE REVIEW

File: precompiles/xcm/src/lib.rs [hunk 1]
```diff
@@ -82,8 +82,11 @@ where
                     .flatten()
             })
             .collect();
-        let amounts: Vec<u128> = input
-            .read::<Vec<U256>>()?
+        let amounts_raw = input.read::<Vec<U256>>()?;
+        if amounts_raw.any(|x| x > u128::max_value().into()) {
+            return Err(revert("Asset amount is too big"))
+        }
```
Comment (Dinonard) [inline precompiles/xcm/src/lib.rs]: Neat!

**Review by Dinonard — APPROVED** [review]
LGTM

**Comment by Dinonard** [comment]
Please just fix the formatting

**Comment by akru** [comment]
@Dinonard thank you for review, done

**Pull request merged**